### DISCUSSION TITLE
Select-all option in survey-multi-select

### DISCUSSION THREAD

**Discussion by Kedersha** [jspsych/jsPsych#2553] [Q&A]
Hello all! 

I'm supporting some researchers who are doing an online version of a vocabulary questionnaire for parents to fill out about their kids. We're using jsPsych 6.3's survey-multi-select plugin specifically. The questionnaire is very long, and for older kids who know a lot of the relevant words it means a *lot* of clicking. 

Has anyone already built a survey-multi-select with a "select all" option? I tried adapting [these](https://stackoverflow.com/questions/386281/how-to-implement-select-all-check-box-in-html) solutions for a couple minutes but realised I should check here before sinking too many hours in.

Thanks!

**Comment by jodeleeuw** [comment]
Hi @Kedersha 

As far as I know this functionality doesn't exist. If you add it, please consider submitting a pull request! It sounds very useful.
